jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -237,64 +246,60 @@ node {
             checkoutGitRepo(installationGitUrl, installationGitRef, githubCredentialsID)
             ensureEvalsDir()
             releaseConfig = readYaml file: installationManifestFile
-            if (releaseTagVar != "") {
-              componentRelease = releaseConfig[releaseTagVar]
-            } else {
-              componentRelease = releaseConfig[productVersionVar]
-            }
         }
     }
 
+    stage('Fetch Current Integreatly Release') {
+        if (releaseTagVar) {
+            componentRelease = releaseConfig[releaseTagVar]
+        } else {
+            componentRelease = releaseConfig[productVersionVar]
+        }
+        if (!componentRelease) {
+            error "[ERROR] Unable to retrieve current release version!"
+        }
+        println "[INFO] componentRelease = ${componentRelease}"
+    }
```

Comment:
Extracted this into it's own stage (Fetch Current Integreatly Release). Responsible for ensuring the current component release version is found and set as componentRelease.